src/ceph.in

```diff
@@ -1,4 +1,4 @@
-#!@Python3_EXECUTABLE@
+#!@Python3_EXECUTABLE@ -u
```

Comment:
@epuertat i cannot find the analysis in the commit message. but my guess is that you assume that the root cause is python writes to stdout in the buffered mode, so it tends to buffer the output, and write to stdout in a bulk of, for instance, 4096 bytes. and `BrokenPipeError` is raised if the stdout is closed when we try to write to stdout.

but why do you think unbuffered mode is the cure? if jenkins agent is the one who consumes the stdout, and it closes the pipe when ceph CLI is still trying to write to it. how is the unbuffered mode help?

shall we instead catch the `BrokenPipeError` exception as suggested by https://github.com/python/cpython/commit/7b0ed43af55c1e2844aa0ccd5e088b2ddd38dbdb ?

Reply:
You're right. I assumed the issue was with the buffer length, since there was a 64  kB limit in older kernels (now it's ~1 MB) and current output is ~80 kB. And I ignored the case of pipe closure. Updating. Thanks @tchaikov !